### DISCUSSION TITLE
Sharing jobRecords at group level

### DISCUSSION THREAD

**Discussion by sgoldman101** [ld32/smarterSlurm#24] [General]
My group shares a pipeline with another group.  To help build up a comprehensive set of jobRecords for each of the tools used in the pipeline can smartSlurm be set up to use a common jobRecords file for all members or a group?  How about for groups of groups?  Could members of Group A and Group B all pull from jobRecordsX to run PipelineX?

**Comment by ld32** [comment]
Sure. Added a variable to https://github.com/ld32/smartSlurm/blob/main/config/config.txt
jobRecordDir=~/.smartSlurm
Group admins can change to any path they want. 

If users prefer a private path, they can copy that config file to their $HOME: 
cp config.txt $HOME/.smartSlurm/
Then edit the variable to any path they want. 

ssbbatch will use the $HOME copy if exist:
if [ -f $HOME/.smartSlurm/config.txt ]; then 
    source $HOME/.smartSlurm/config.txt
else     
    source $(dirname $0)/../config/config.txt || { echo Config list file not found: config.txt; exit 1; }
fi

**Comment by sgoldman101** [comment]
That's great! I'll let my group admin know it can be set up like this and we'll test it out.  Thanks!
